### PR TITLE
Feature/responsive css

### PR DESCRIPTION
**Pull request by ericli1996**
Module 3: Stretch Project PR: Breathe Easy

_What iteration is being worked on?_

Responsive CSS Design for mobile view.

_What is in request and being accomplished?_

Added media queries for components, fixed bug as per your suggestions, that was generating a TypeScript error after a user selected State, City, and hit submit.

Collaborators: @hheyhhay @ericli1996 @williamphelps13

Other information:

Cannot seem to get this background image to scale with added child elements. I tried `overflow: auto` as well as some other things, but I think it has to do with the .backdrop and .darken-backdrop classes that control the background image and opacity, having position relative/absolute. So it seems that the elements over this background are not actually contained inside it. 

Also, I tried adding required to the form select elements in order to disable the 'Show AQI' button until a user populates both state and city fields. However, since the the button is now outside the form element, these built-in attributes do not apply. In addition, the Router Link is being clicked, which tries to render a SelectedCity component from our empty state in the App component. So to move forward, I think the best plan of attack would be to conditionally disable this Link.

### CODE REVIEW

File: src/Components/App/App.css [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
+  height: max-content;
 }
 
 .backdrop {
+  height: 100%;
   width: 100vw;
-  height: 100vh;
   position: relative;
   z-index: -1;
   opacity: 80%;
```
Comment (hheyhhay) [inline src/Components/App/App.css:11]: This opacity needs to be  changed to a number

File: src/Components/App/App.css [hunk 2]
```diff
@@ -39,7 +47,7 @@ main {
 }
 
 .selected-city-shading {
-  height: 100vh;
+  min-height: 100vh;
   width: 300px;
   opacity: 85%;
```
Comment (hheyhhay) [inline src/Components/App/App.css:52]: Needs to be covereted to a number

File: src/Components/OtherCities/OtherCities.tsx [hunk 3]
```diff
@@ -28,9 +28,10 @@ const OtherCities: React.FC<Props> = ({ otherCitiesData, deleteCityData, selecte
   })
 
   return (
-    <section className='other-cities-container'>
-      {otherCityCards}
-    </section>
+    <div>
+      {otherCityCards.length > 0 &&
```
Comment (hheyhhay) [inline src/Components/OtherCities/OtherCities.tsx:32]: otherCityCards is a truthy statement so we shouldn't need the .length > 0

**Review by williamphelps13 — APPROVED** [review]
A few minor changes:
- Should we add the heading we have in the mock-up?
- The green of good aqi needs a bit more contrast
- When two cities have the same aqi should the text be black?
- The opacity overlay over the background doesn't go to the bottom
- In mobile the background doesn't go to the bottom

**Pull request merged**